proposals/description/merger.ts

```diff
@@ -40,7 +40,16 @@ const merger: ProposalDescription = {
     }
   ],
   description: `
-  Code: 
+  This proposal represents the Tribe half of the FeiRari merger code. It executes the following steps:
+  1. Accept PegExchanger contract for swapping RGT to TRIBE at ~26.7 TRIBE per RGT exchange rate
+  2. Accept TRIBERagequit contract
+  3. Seed PegExchanger with 270m TRIBE
+  4. Grant FEI minting to TRIBERagequit
```

Comment:
A safer pattern would be to mint FEI to the TRIBERagequit on a dripper rather than giving that contract minting abilities.

Reply:
It's too late to change this; the merger is already up for vote on Rari.